traitsui/examples/demo/Extras/Image_editor_demo.py

```diff
@@ -62,18 +65,34 @@ class Employee(HasTraits):
                     ),
```

Comment:
I was surprised that this worked previously without defining the picture trait on the class - just passing `picture=ImageResource('e-logo-rev', search_path=search_path)` on instantiation.